docs/grammar.md

```diff
@@ -15,9 +15,11 @@ Thanks to ANTLR, APM has new layer of abstraction, which solves these and other
 ### Definitions
 ```
 DEFINE baseGroup 'group-base'
+DEFINE encryptedBaseGroup '{284f457a09a3d7f31fbc078343bb93179647297075237025786edd28ece8b919}' # encrypted string

```

Comment:
Is it possible to set encrypted password for user like this?
CREATE-USER "user1" path= "/home/users/myusers" BEGIN
    SET-PASSWORD '{5d81e5b2e99bbb06ce921355ba4b9d2bde49723712ba58b5b6cfbd93859c34a9}'
END